operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -16,13 +11,13 @@ powerscale:
     replication: "v1.10.0"
     observability: "v1.10.0"
     resiliency: "v1.11.0"
+  v2.13.0:
+    authorization: "v1.13.0"
```

Comment:
@atye @shaynafinocchiaro can you please review and confirm on the auth related versions?

Reply:
I believe we want to use 2.1.0 since Auth 1.x is being deprecated soon

Reply:
Discussed with @shaynafinocchiaro, this can be v2.1.0 since we want to start having users move to Authorization v2. This also means that the driver samples for CSM 1.13 should use v2.1.0 for the Authorization sidecar.

Reply:
updated, please review